packages/firebase_crashlytics/firebase_crashlytics_platform_interface/lib/src/method_channel/method_channel_crashlytics.dart

```diff
@@ -126,13 +125,13 @@ class MethodChannelFirebaseCrashlytics extends FirebaseCrashlyticsPlatform {
   @override
   Future<void> setCrashlyticsCollectionEnabled(bool enabled) async {
     try {
-      Map<String, dynamic> data = await channel
+      Map<String, dynamic>? data = await channel
           .invokeMapMethod<String, dynamic>(
               'Crashlytics#setCrashlyticsCollectionEnabled', <String, dynamic>{
         'enabled': enabled,
       });
 
-      _isCrashlyticsCollectionEnabled = data['isCrashlyticsCollectionEnabled'];
+      _isCrashlyticsCollectionEnabled = data!['isCrashlyticsCollectionEnabled'];
     } catch (e, s) {
```

Comment:
We probably should instead do:

```dart
} on PlatformException catch (e, s) {
  throw  platformExceptionToFirebaseException(e, s);
}
```

and remove this `convertPlatformException`